lib/iphone-simulator-xcode-7.ts

```diff
@@ -93,10 +94,15 @@ export class XCode7Simulator extends IPhoneSimulatorNameGetter implements ISimul
 	}
 
 	public stopApplication(deviceId: string, cfBundleExecutable: string): IFuture<string> {
-		try {
-			return childProcess.exec(`killall ${cfBundleExecutable}`, {skipError: true});
-		} catch(e) {
-		}
+		return (() => {
+			let cmdline = "ps cax|awk \"\\$3!=\\\"Z\\\"{print \\$1, \\$5}\"|grep " + cfBundleExecutable;
```

Comment:
Can you revert this change? As we've discussed `killall` was not working as the debugger is not allowing the process to die.